go.mod

```diff
@@ -340,6 +351,8 @@ require (
 	mvdan.cc/interfacer v0.0.0-20180901003855-c20040233aed // indirect
 	mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
 	mvdan.cc/unparam v0.0.0-20220706161116-678bad134442 // indirect
+	sigs.k8s.io/cluster-api-provider-aws v0.0.0-00010101000000-000000000000 // indirect
+	sigs.k8s.io/cluster-api-provider-azure v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
Curious why it's adding cluster-api-provider-azure and cluster-api-provider-aws as indirect. Aren't threse already in added as replacements pointing to the openshift forks?